src/views/product-landing/server.ts

```diff
@@ -1,13 +1,23 @@
 import fs from 'fs'
 import path from 'path'
 import slugify from 'slugify'
-import { Product } from 'types/products'
+import { MenuItem } from 'components/sidebar'
+import { ProductSlug } from 'types/products'
+
+export type LandingPageProduct = {
+  name: string
+  sidebar: {
+    landingPageNavData: MenuItem[]
+    resourcesNavData: MenuItem[]
+  }
+  slug: ProductSlug
+}
```

Comment:
Since this is a complex issue that will require some exploration and discussion, I think we should stick with using `Product` for now and defer breaking up that interface in a future PR.

Reply:
@zchsh this still needs addressed. 👆🏻 Could you undo these changes and we can work on improving the TypeScript in a future PR? Agree that it needs improvement but I think it needs more time and attention as its own task.

Reply:
@ashleemboyer For sure, will use `Product` for now 👍

Reply:
✅ Updated